machine_management/adding-rhel-compute.adoc

```diff
@@ -28,9 +28,9 @@ include::modules/rhel-preparing-node.adoc[leveloffset=+1]
 include::modules/rhel-attaching-instance-aws.adoc[leveloffset=+1]
 
 .Additional resources
-* See xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[Required AWS permissions for IAM roles]
+* See xref:../installing/installing_aws/installing-aws-account.adoc#installation-aws-permissions-iam-roles_installing-aws-account[Required AWS permissions for IAM roles]
```

Comment:
I realize that you only updated the link, but I think we want this bullet to end with a period.  Do you mind adding that?